StortSpelProjekt/Game/src/main.cpp

```diff
@@ -1,5 +1,7 @@
 #include "Engine.h"
 #include "Components/PlayerInputComponent.h"
+#include "ImGUI/ImGuiHandler.h"
```

Comment:
Should this be included here? My thought was that the only thing we need to include here is "engine.h" and "game.h" later on?